### DISCUSSION TITLE
New spawnmob permission for using data

### DISCUSSION THREAD

**Discussion by baailey** [EssentialsX/Essentials#4270] [Ideas and Feature Suggestions] [answered]
Hello,

Truthfully I'm not too confident on whether or not this will be accepted, but gonna give it a try anyway.

If a user has access to `essentials.spawnmob.zombie` for example, they unlock full access to spawning baby zombies, zombies in full diamond gear etc. I would like for the data aspect of spawnmob to only be usable if they have access to the permission `essentials.spawnmob.zombie.data` (change however you wish). 

Having full access to data just because you have spawnmob is something I feel is a bit unnecessary. The amount of changes you can make to anything spawnmob related is very limited on Ess, so having more options available for this command can allow for more customisation on servers, and allow granting access to certain mobs without the worry of any additional data being added to a spawnmob command.

**Comment by pop4959** [comment, marked as answer]
I don't think this is something we will be changing since doing so would require re-evaluating how permissions are handled for the command (breaking everyone's existing perms) and this was also mostly designed for use by server admins, and not players or others with limited permissions.
